app/src/app/providers.tsx

```diff
@@ -32,7 +35,14 @@ export function Providers({ children }: { children: React.ReactNode }) {
             accentColor: "#22CC02",
           })}
         >
-          <ApolloProvider client={apolloClient}>{children}</ApolloProvider>
+          <ApolloProvider client={apolloClient}>
+            <OnchainKitProvider
+              apiKey={NEXT_PUBLIC_COINBASE_API_KEY}
```

Comment:
Is this api key really required?

Reply:
I've checked that just fetching ENS and verification status works without an API key. I'll remove it.